python/tests/test_dash_component.py

```
@@ -17,7 +17,7 @@ def test_dash_bundle_is_packaged():
 
 def test_dashmap_requires_dash_or_has_component():
     if dmod.Component is None:
-        with pytest.raises(ImportError, match=r"geolibre\[dash\]"):
+        with pytest.raises(ImportError, match="geolibre\[dash\]"):
```

Ruff (0.16.3)

[warning] Invalid escape sequence: `\[`

Use a raw string literal

(W605)